engine/api/mailer/MockMailer.php

```diff
@@ -8,20 +8,32 @@ class MockMailer extends Mailer
     function __construct()
     {
         $this->mockedMail = new MockedEmail();
+        $this->global_address = "test@test.fr";
+        $this->mockedMail->addresses = [];
     }
 
     function createEmail($address, $subject, $content): self
     {
-        $this->mockedMail->address = $address;
+        $this->mockedMail->addresses[] = $address;
         $this->mockedMail->subject = $subject;
         $this->mockedMail->content = $content;
         return $this;
     }
 
+    function createBulkEmails($addresses, $subject, $content): self
+    {
+        foreach ($addresses as $address => $name) {
+            array_push($this->mockedMail->addresses, $address);
+        }
```

Comment:
You can just directly assign here I think, or use array_merge

```suggestion
        $this->mockedMail->addresses = $addresses;
```

Reply:
Just did that, thx